lb_content_resolver/content_resolver.py

```diff
@@ -190,12 +197,29 @@ def add_file_to_index(self, relative_path, format, mtime):
             mdata["mtime"] = mtime
             mdata["file_path"] = file_path
 
-            # now add the record
-            self.index_metadata(mdata)
+            mdata["artist_mbid"] = self.convert_to_uuid(mdata["artist_mbid"])
+            mdata["release_mbid"] = self.convert_to_uuid(mdata["release_mbid"])
+            mdata["recording_mbid"] = self.convert_to_uuid(mdata["recording_mbid"])
+
+            # now add/update the record
+            return self.add_or_update_recording(mdata)
+        return "error"
```

Comment:
Wasn't sure what I should return in this case but "error" felt the closest and it will be counted as such back in the caller.

Reply:
That's fine. :)